zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -449,7 +450,12 @@ ascan.options.strength.high.label = (More attacks)
 ascan.options.strength.insane   = Insane
 ascan.options.strength.insane.label = (MUCH more attacks)
 ascan.options.strengthTo.label		= Strength To
-ascan.options.thresholdTo.label		= Threshold To 
+ascan.options.thresholdTo.label		= Threshold To
+
+# Filtering Requests from Scanner Reasons
```

Comment:
Not necessary.

Reply:
you are saying this for the comment or about the messages ?

Reply:
The comment.